lib/core/src/lib/form/components/widgets/error/error.component.scss

```diff
@@ -1,5 +1,6 @@
 .adf-error {
     display: flex;
+    align-items: center;
```

Comment:
is this expected? the error under each field should be centered, not left-aligned?